packages/module/src/CodeModal/CodeModal.tsx

```diff
@@ -82,15 +86,25 @@ export const CodeModal: React.FunctionComponent<CodeModalProps> = ({
     }
   };
 
-  return (
+  /* eslint-disable indent */
+  const getHeight = (displayMode: ChatbotDisplayMode) => {
+    switch (displayMode) {
+      case ChatbotDisplayMode.docked:
+        return '100vh';
+      default:
+        return '45vh';
+    }
```

Comment:
This is a temporary workaround since the CodeEditor is acting like it requires a height. I would love it if I could just assume it took up 100% of the parent, but it's not cooperating.